subsys/bluetooth/audio/Kconfig.tbs

```diff
@@ -140,21 +140,23 @@ endif # BT_TBS
 
 ##################### Call Control Client #####################
 
-config BT_TBS_CLIENT
+config BT_TBS_CLIENT_TBS
```

Comment:
Hmm, in TBS the server side shall support GTBS, and then 0 or more TBS instances. 

Should that same requirement go for the TBS client? 

In other words, rather than adding this, should `CONFIG_BT_TBS_CLIENT_MAX_TBS_INSTANCES` (if not already) only refer to TBS instances, and not GTBS, so that if `CONFIG_BT_TBS_CLIENT_MAX_TBS_INSTANCES == 0`, then we effectively have a GTBS-only client?

Reply:
Nice idea, I did not know that `range` can be conditional as well. Could you please take a look at latest revision?

Reply:
@Thalley I had to fix the `primary_discover_func` as well, because the discovery was not stopped even if GTBS was found and there was no space left for TBS.